migrations/20200707221810-mandatory-patient-fields.js

```diff
@@ -0,0 +1,51 @@
+"use strict";
+
+module.exports = {
+  up: (queryInterface, Sequelize) => {
+    return queryInterface.sequelize.transaction((t) => {
+      return Promise.all([
+        queryInterface.changeColumn("patients", "barcodeValue", {
+          type: Sequelize.BIGINT,
+          allowNull: false,
+        }),
+        queryInterface.sequelize.query(
```

Comment:
you're right that this is the best way to do this, i truly hate sequelize enums with a passion now